message/abandon.go

```diff
@@ -20,7 +20,7 @@ func (a *abandon) Do(ctx context.Context, _ Handler, message *servicebus.Message
 	defer span.End()
 
 	if err := message.Abandon(ctx); err != nil {
-		tab.For(ctx).Error(err)
+		span.AddAttributes(tab.StringAttribute("eventMessage", err.Error()), tab.StringAttribute("eventLevel", "error"))
```

Comment:
So after refreshing my understanding of opencensus/opentracing I don't think they're made to get you logs. Both have become open telemetry and get you call stack tracing and metrics. 
But you need an exporter or tracer like jaeger or zipkin

Reply:
opentelemetry (successort to both of thes has an stdout exporetr
https://pkg.go.dev/go.opentelemetry.io/otel/exporters/stdout
https://github.com/open-telemetry/opentelemetry-go/tree/master/exporters

Not sure if this custom exporter for opencensus would actually get you logging
https://opencensus.io/exporters/custom-exporter/go/trace/

Reply:
https://play.golang.org/p/3LrQkCX46E5
Yeah this doesn't get you logging but function tracing. Might want to actually talk to david justice.

Reply:
Here we just collect data and store into spans. The caller of this library can decide how to export and integrate with the logging system. It might not be necessary to worry about export here.

Reply:
Harry pointed me to our kusto exporter which answered a lot of questions.

Still nto sure why .For(ctx) wouldn't aslo append attributes. We're going to try and follow up with @David Justice about that.